src/main.js

```diff
@@ -1,4 +1,28 @@
 import axios from 'axios';
+
+
+import { getTasks, displayTask } from './init';
+
+window.deleteTask = deleteTask;
+window.updateTask = updateTask;
+window.editTask = editTask;
```

Comment:
Try to find other alternatives for this, polluting the global window object is not a great practice.